lib/rouster.rb

```diff
@@ -704,7 +732,7 @@ def restart(wait=nil, expected_exitcodes = [0])
   # (should be) private method that executes commands on the local host (not guest VM)
   #
   # returns STDOUT|STDERR, raises Rouster::LocalExecutionError on non 0 exit code
-  # sets @exitcode
+  # pushes to @ssh_exitcode
```

Comment:
i know this is legacy, but now it feels even weirder - this is not an SSH exit code, it's a local run exit code. given the use cases for this method (all local, and mostly related to environment sanity), how do you feel about just not returning the exitcode - if it isn't 0, we're throwing anyway, and if it is, well, that's just expected.